package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/mendix-hybrid-app-template",
-  "version": "6.0.3",
+  "version": "7.0.0",
```

Comment:
what is the reason of increasing a major version?

Reply:
The Android SDK upgrade is a significant change for Cordova.